src/App.tsx

```diff
@@ -1,14 +1,78 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { Todo } from './types/Todo';
+import { FilterOptions, FilterType } from './types/FilterType';
+
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { ErrorText } from './components/ErrorText';
+import { getTodos } from './api';
+import { getFilteredTodos } from './helpers/getFilteredTodos';
+
+const initialFilterValue: FilterType = {
+  filterByTitle: '',
+  filterByStatus: FilterOptions.All,
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isVisibleModal, setIsVisibleModal] = useState(false);
```

Comment:
You don't really need this state, you can show your modal if `activeTodo` is not `null`